dspace-api/pom.xml

```diff
@@ -869,6 +886,12 @@
             <artifactId>mockserver-junit-rule</artifactId>
             <version>5.11.2</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
I suspect this exclusion was to solve conflicts with a different dependency.  However, in that scenario, we tend to add a comment which says something like:
`<!-- Exclude snakeyaml to avoid conflicts with ____ -->`  (where the blank should be the other dependency which is conflicting here).

So, I agree with Raf it'd be nice to add a small note to this exclusion as to why it's necessary.

Reply:
this has been addressed here https://github.com/DSpace/DSpace/pull/8271/commits/2aff1f610cebc4d543acb7b04a1ba8b4e8bbba22